pallets/farming/src/lib.rs

```diff
@@ -388,6 +398,29 @@ pub mod pallet {
     #[pallet::storage]
     pub type PoolFarmers<T: Config> =
         StorageMap<_, Identity, T::AccountId, Vec<PoolFarmer<T>>, ValueQuery>;
+
+    #[pallet::storage]
+    #[pallet::getter(fn lp_min_xor_for_bonus_reward)]
+    pub type LpMinXorForBonusReward<T: Config> = StorageValue<_, Balance, ValueQuery>;
+
+    #[pallet::call]
+    impl<T: Config> Pallet<T> {
+        #[pallet::call_index(0)]
+        #[pallet::weight(<T as Config>::WeightInfo::set_lp_min_xor_for_bonus_reward())]
+        pub fn set_lp_min_xor_for_bonus_reward(
+            origin: OriginFor<T>,
+            new_lp_min_xor_for_bonus_reward: Balance,
```

Comment:
Do we need to check `new_lp_min_xor_for_bonus_reward` value somehow? Is `new_lp_min_xor_for_bonus_reward = 0` ok?

Reply:
I added a check with a subjective error